src/runtime/server.ts

```diff
@@ -44,15 +47,18 @@ global["browser"] = false;
 
 class HTTPRuntimeClient extends RuntimeClient {
   server: ServerDatabase;
-  constructor() {
+  constructor(databases) {
     let server = new ServerDatabase();
     const dbs = {
       "http": new HttpDatabase(),
       "server": server,
       "shared": shared,
       "browser": new Database(),
     }
-    super(dbs);
+    var uniondb = {}
```

Comment:
These `var`s should be `let`s in TS land. `let` gives us sane scoping rules. You really don't want to think about the shadowing BS var does.